<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginIFDemo_Main</class>
 <widget class="QDialog" name="PluginIFDemo_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabInterfaceTypes">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabBoolean">
      <attribute name="title">
       <string>Boolean</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="chkTestCheckbox">
         <property name="text">
          <string>Check the value in the toolbar checkbox</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabButton">
      <attribute name="title">
       <string>Button</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
         <item>
          <widget class="QLineEdit" name="txtButtonName">
           <property name="placeholderText">
            <string>Text to be sent to toolbar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendButtonText">
           <property name="text">
            <string>Send Text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabString">
      <attribute name="title">
       <string>String</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <widget class="QLineEdit" name="txtEdit">
           <property name="placeholderText">
            <string>Text to be sent to toolbar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendText">
           <property name="text">
            <string>Send Text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSelector">
      <attribute name="title">
       <string>Selector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Display:</string>
           </property>
           <property name="buddy">
            <cstring>txtNewItemDisplay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtNewItemDisplay">
           <property name="placeholderText">
            <string>Enter display text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Value:</string>
           </property>
           <property name="buddy">
            <cstring>txtNewItemValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtNewItemValue">
           <property name="placeholderText">
            <string>Enter value text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddItem">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListView" name="lstItems"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btnRemoveItem">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSendList">
             <property name="text">
              <string>Send List</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAddRemoveImmediate">
         <property name="text">
          <string>Add and remove will immediately be send to interface</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkSendSelect">
         <property name="text">
          <string>Update interface with selection</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Update</string>
           </property>
           <property name="buddy">
            <cstring>cmbEntryIndex</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbEntryIndex"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>with</string>
           </property>
           <property name="buddy">
            <cstring>txtUpdateDisplayValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtUpdateDisplayValue">
           <property name="placeholderText">
            <string>Enter value text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendUpdateItem">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Element to be updated</string>
       </property>
       <property name="buddy">
        <cstring>cmbElements</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnEnable">
       <property name="text">
        <string>Enable element</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnDisable">
       <property name="text">
        <string>Disable element</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cmbElements"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSilent">
     <property name="text">
      <string>Activate updates silently</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="logView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
